plugin.go

```diff
@@ -16,10 +16,13 @@ type DatasourcePluginImpl struct {
 }
 
 func (p *DatasourcePluginImpl) GRPCServer(broker *plugin.GRPCBroker, s *grpc.Server) error {
```

Comment:
These need comments. Same with wrappers etc - kept getting (and maybe still am confused) about the relationships.